content/zh/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,363 @@
+to override this behaviour, see [Delete owner objects and orphan dependents](/docs/tasks/administer-cluster/use-cascading-deletion/#set-orphan-deletion-policy).
+-->
+### 被遗弃的依赖对象    {#orphaned-dependents}
+
+当 Kubernetes 删除某个属主对象时，被留下来的依赖对象被称作*被遗弃的（Orphaned）*对象。
+默认情况下，Kubernetes 会删除依赖对象。要了解如何重载这种默认行为，可参阅
+[删除属主对象和遗弃依赖对象](/zh/docs/tasks/administer-cluster/use-cascading-deletion/#set-orphan-deletion-policy).
+
+<!--
+## Garbage collection of unused containers and images {#containers-images}
+
+The {{<glossary_tooltip text="kubelet" term_id="kubelet">}} performs garbage
+collection on unused images every five minutes and on unused containers every
+minute. You should avoid using external garbage collection tools, as these can
+break the kubelet behavior and remove containers that should exist. 
+-->
+## 未使用容器和镜像的垃圾收集     {#containers-images}
+
+{{<glossary_tooltip text="kubelet" term_id="kubelet">}} 会每五分钟对未使用的镜像执行一次垃圾收集，
+每分钟对未使用的容器执行一次垃圾收集。
+你应该避免使用外部的垃圾收集工具，因为外部工具可能会破坏 kubelet
+的行为，移除应该保留的容器。
+
+<!--
+To configure options for unused container and image garbage collection, tune the
+kubelet using a [configuration file](/docs/tasks/administer-cluster/kubelet-config-file/)
+and change the parameters related to garbage collection using the
+[`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration)
+resource type.
+-->
+要配置对未使用容器和镜像的垃圾收集选项，可以使用一个
+[配置文件](/zh/docs/tasks/administer-cluster/kubelet-config-file/)，基于
+[`KubeletConfiguration`](/zh/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) 
+资源类型来调整与垃圾搜集相关的 kubelet 行为。
+
+<!--
+### Container image lifecycle
+
+Kubernetes manages the lifecycle of all images through its *image manager*,
+which is part of the kubelet, with the cooperation of 
+{{< glossary_tooltip text="cadvisor" term_id="cadvisor" >}}. The kubelet
+considers the following disk usage limits when making garbage collection
+decisions:
+-->
+### 容器镜像生命期     {#container-image-lifecycle}
+
+Kubernetes 通过其*镜像管理器（Image Manager）*来管理所有镜像的生命周期，
+该管理器是 kubelet 的一部分，工作时与
+{{< glossary_tooltip text="cadvisor" term_id="cadvisor" >}} 协同。
+kubelet 在作出垃圾收集决定时会考虑如下磁盘用量约束：
+
+* `HighThresholdPercent`
+* `LowThresholdPercent`
+
+<!--
+Disk usage above the configured `HighThresholdPercent` value triggers garbage
+collection, which deletes images in order based on the last time they were used,
+starting with the oldest first. The kubelet deletes images
+until disk usage reaches the `LowThresholdPercent` value.
+-->
+磁盘用量超出所配置的 `HighThresholdPercent` 值时会触发垃圾收集，
+垃圾收集器会基于镜像上次被使用的时间来按顺序删除它们，首先删除的是最老的镜像。
+kubelet 会持续删除镜像，直到磁盘用量到达 `LowThresholdPercent` 值为止。
+
+<!--
+### Container image garbage collection {#container-image-garbage-collection}
+
+The kubelet garbage collects unused containers based on the following variables,
+which you can define: 
+-->
+### 容器镜像垃圾收集    {#container-image-garbage-collection}
```

Comment:
This now needs updating as the upstream (English) text has changed - thanks @tengqm!

Reply:
right. Thanks for the reminder.